google-cloud-testing/storage-benchwrapper/README.md

```diff
@@ -0,0 +1,67 @@
+# storage-benchwrapper
+
+storage-benchwrapper is a gRPC wrapper around the storage library for benchmarking purposes.
+
+## Running
+
+```
+mvn clean install -DskipTests=true (one time)
+cd google-cloud-testing/storage-benchwrapper
+export STORAGE_EMULATOR_HOST=localhost:8080
+mvn clean install exec:java -DskipTests=true -Dport=8081
+```
+
+## Generating .proto sources
+
+Sources are generated automatically as part of the protobuf-maven-plugin plugin,
```

Comment:
Try to avoid passive voice.
"automatically" is unneeded since it's implied.

Reply:
Done